tracer/src/Datadog.Trace.Tools.Runner/Utils.cs

```diff
@@ -280,5 +284,44 @@ public static string GetEnvironmentVariable(string key, string defaultValue = nu
 
             return defaultValue;
         }
+
+        public static async Task<bool> CheckAgentConnectionAsync(string agentUrl)
+        {
+            var env = new NameValueCollection();
+            if (!string.IsNullOrWhiteSpace(agentUrl))
+            {
+                env["DD_TRACE_AGENT_URL"] = agentUrl;
+            }
+
+            var globalSettings = GlobalSettings.CreateDefaultConfigurationSource();
+            globalSettings.Add(new NameValueConfigurationSource(env));
+            var tracerSettings = new TracerSettings(globalSettings);
+            var agentWriter = new CIAgentWriter(tracerSettings);
+
+            try
+            {
+                if (await agentWriter.Ping().ConfigureAwait(false))
+                {
+                    Console.WriteLine($"Connection with the Datadog Agent at {tracerSettings.AgentUri} was successful.");
```

Comment:
Do we definitely want to write to the console when the check was successful? This could break people's logs if they're being monitored I think? 

It seems ok (and required) to write to the console if we're going to fail anyway, but not if we're going to continue.

Reply:
makes sense, done in 99559cc